### PR TITLE
feat: fix memo len

### PR DESCRIPTION
**Pull request by mateuszaaa**
## Summary by CodeRabbit

* **Bug Fixes**
  * Improved handling of refunds for large multi-token transfers at the refund-log size boundary.
  * Ensured sender balances are correctly restored when refund processing reaches the supported log-size limit.
  * Added safeguards for transfers involving many tokens, including cases that exceed the available refund-log capacity.
  * Added regression coverage for boundary conditions to help prevent future refund-processing failures.

### CODE REVIEW

File: tests/src/tests/defuse/tokens/nep245/mt_transfer_resolve_gas.rs [hunk 1]
```diff
@@ -408,6 +408,113 @@ async fn mt_batch_transfer_call_rejects_transfer_when_refund_log_exceeds_limit(
...
+                approvals: None,
+                memo: None,
+                msg: &serde_json::to_string(&MTReceiverMode::RefundAll).unwrap(),
+            })
+            .deposit(NearToken::from_yoctonear(1))
+            .gas(Gas::from_tgas(300)),
+        )
+        .wait_until::<Final>()
+        .await
+        .unwrap();
```
[the remark this answers is not in the document]
Comment (mateuszaaa) [reply tests/src/tests/defuse/tokens/nep245/mt_transfer_resolve_gas.rs:501]: false positive, i manually inspect the results belowe

File: tests/src/tests/defuse/tokens/nep245/mt_transfer_resolve_gas.rs [hunk 2]
```diff
@@ -408,6 +411,125 @@ async fn mt_batch_transfer_call_rejects_transfer_when_refund_log_exceeds_limit(
     assert_eq!(balance_after, balance_before,);
 }
 
+const REPRO_TOKEN_COUNT: usize = 66;
+const REPRO_FIRST_TOKEN_ID_LEN: usize = MAX_TOKEN_ID_LEN - 8;
```
Comment (hlgltvnnk) [inline tests/src/tests/defuse/tokens/nep245/mt_transfer_resolve_gas.rs:415]: please leave comment explaining why 8 (total log len 1 token * MAX_TOKEN_ID_LEN - TOTAL_LOG_LENGTH_LIMIT)

**Review by mitinarseny — DISMISSED** [review]

**Review by mitinarseny — APPROVED** [review]
